app/assets/stylesheets/providers/organisations.scss

```diff
@@ -0,0 +1,32 @@
+.organisation-item {
```

Comment:
ditto this...

could it not be renamed as search-result-item and used for proceedings _and_ organisations?

Reply:
i guess it could but the javascript for both would need to reflect the changes too, or alternatively, be added as an additional class just for the css.

Reply:
myself and peter have moved these into a shared scss file now